src/components/ScanQRModal/ScanQRWrapper/index.tsx

```diff
@@ -1,20 +1,23 @@
+import { Button, Icon } from '@gnosis.pm/safe-react-components'
 import { makeStyles } from '@material-ui/core/styles'
-import { useState } from 'react'
-import * as React from 'react'
+import React from 'react'
```

Comment:
I usually prefer to have `import React, { useState }` so is explicit that we use useState

Reply:
Oh, I prefer the most verbose/explicit way: `React.anyHook`, so it's easier to follow while reading the code if it's a React's hook or a different one. Whichever the final decision is, this should be added and enforced in a style-guide for sure.